perf/main.py

```diff
@@ -0,0 +1,19 @@
+from locust import HttpUser, constant_pacing, task
```

Comment:
I like locust though we are never going to have millions of users and our performance (where load is concerned) is easily loaded via a set of curl scripts (or using osidb bindings). I am not sure we want to embed perf tests directly in the repo itself ... I also think perf is heavily dependent on total env (eg. including network) and maybe such script would live in ops repo.

Reply:
I don't see why not, dev dependencies are not installed in the production image, this provides a single way to define performance tests / benchmarking which reduces a lot of the risk of having hand-rolled solutions (different results from different people), and you can run these tests against stage/production directly and it's easy to integrate into various pipelines

Reply:
Sure, but there is cognitive load adding any dependency... also I do not know what locusts gives us (or why we would need it) beyond what one can achieve using simpler methods ... for example with curl we can easily perform parallel loading with something like: 

```
curl "https://localhost:8010/osidb/api/v1/affects?offset=[1-100]"  -w "%{time_connect},%{time_total},%{speed_download},%{http_code},%{size_download},%{url_effective}\n" -o /dev/null -Z
```
this has the added benefit that the perf test is not executed in python ( variation is very useful in testing in general).

Alternately we might emulate how we have done this in [component-registry](https://github.com/RedHatProductSecurity/component-registry/blob/main/tests/test_performance.py) using plain ole pytest (which also uses python-request as well ... if u want that http1.1 constraint;))

The tests you have written seem to only scratch the surface of locust capability which makes me wonder if we need it.